enclave/sgx/collateralinfo.c

```diff
@@ -131,8 +236,8 @@ OE_WEAK_ALIAS(
  * @param[in] default_sizes The default quote verification collateral sizes.
  */
 void oe_prealloc_quote_verification_collateral_args(
-    oe_get_sgx_quote_verification_collateral_args_t* buf,
-    oe_get_sgx_quote_verification_collateral_args_t* default_sizes);
+    oe_get_sgx_quote_verification_collateral_with_params_args_t* buf,
```

Comment:
Not new in this PR so ok to leave if you want, but "buf" violates [OE coding style](https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md).

Reply:
I would leave it as it was, there are lots of places need to be updated, better to be in a separated PR